src/main/java/org/concordion/api/Skipped.java

```diff
@@ -0,0 +1,8 @@
+package org.concordion.api;
+
+import java.lang.annotation.*;
+
+@java.lang.annotation.Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Skipped {
```

Comment:
If we use the jUnit @Ignore annotation then we won't need this interface. Need to verify that this annotation exists in the most recent version of jUnit tho - no point relying on a dependency that we know will change!